front/src/modules/ui/table/components/TableCell.tsx

```diff
@@ -1,18 +1,32 @@
-import { PropsWithChildren } from 'react';
+import { ReactNode } from 'react';
 import styled from '@emotion/styled';
 
-const StyledTableCell = styled.td`
-  padding: 0 ${({ theme }) => theme.spacing(2)};
-`;
+type TableCellProps = {
+  children?: ReactNode;
+  className?: string;
+};
 
-const StyledTableCellContent = styled.div`
+const StyledTableCell = styled.div<{ isNumberCell: boolean }>`
   align-items: center;
+  color: ${({ theme }) => theme.font.color.secondary};
   display: flex;
   height: ${({ theme }) => theme.spacing(8)};
+  justify-content: ${({ isNumberCell }) =>
+    isNumberCell ? 'flex-end' : 'flex-start'};
+  padding: 0 ${({ theme }) => theme.spacing(2)};
+  text-align: ${({ isNumberCell }) => (isNumberCell ? 'right' : 'left')};
 `;
 
-export const TableCell = ({ children }: PropsWithChildren) => (
-  <StyledTableCell>
-    <StyledTableCellContent>{children}</StyledTableCellContent>
-  </StyledTableCell>
-);
+export const TableCell = ({ children, className }: TableCellProps) => {
+  return (
+    <StyledTableCell
+      className={className}
+      isNumberCell={
+        typeof children === 'number' ||
```

Comment:
Could we use a `isNumber()` typeguard ?

Reply:
I implemented an `align` prop as suggested by @charlesBochet (see https://github.com/twentyhq/twenty/pull/1885#discussion_r1347665751), therefore this check is not needed anymore.